ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+- [ ] I'm reporting a bug, not asking for help
+- [ ] I've looked at the [documentation](http://leafletjs.com/reference-1.0.0.html) to make sure this is not a documented behaviour
```

Comment:
Typo: «not documented behaviour». Maybe it's clearer if the wording is «to make sure the behaviour is documented and expected»